window.c

```diff
@@ -201,6 +203,24 @@ void win_open(win_t *win)
 	if (win->xwin == None)
 		error(EXIT_FAILURE, 0, "Error creating X window");
 
+  // set the _NET_WM_PID
+  pid_t pid = getpid();
+  XChangeProperty(e->dpy, win->xwin,
+                  atoms[ATOM__NET_WM_PID], XA_CARDINAL, sizeof(pid_t) * 8,
+                  PropModeReplace, (unsigned char *) &pid, 1);
+
+  // set the WM_CLIENT_MACHINE
+  char hostname[255];
```

Comment:
Let's declare `pid` and `hostname` on top of the block since that's the existing style on this codebase.

Reply:
moved the declarations atop the function under the other vars